site/content/integrate/apps/functionality/_index.md

```diff
@@ -3,6 +3,16 @@ title: Functionality
 description: Apps API functionality
 weight: 60
 ---
+In addition to the {{<newtabref title="Mattermost REST APIs" href="https://api.mattermost.com">}}, the Apps framework provides additional functionality:
 
-### Coming soon!
-In the future this section will explore additional functionality that Apps provide. For example, event subscriptions.
+- [KV Store]({{< ref "kv-store" >}}) - a simple key-value store for App-specific data.
+
+- [OAuth2 Store]({{< ref "mattermost-api#apps-api" >}}) - store, expand, and retrieve user and App OAuth2 configuration data.
+
+- [Calling other Apps]({{< ref "mattermost-api#apps-api" >}}) - use the Call API to invoke other Apps (experimental).
```

Comment:
Not sure if we should advertise this if it's experimental cc @levb

Reply:
@levb @mickmister How would anyone know if a feature of the Apps framework is "experimental" other than what's in the code or existing docs? Can we not include any experimental stuff in framework releases so this isn't an issue?

Reply:
This potentially could change with the OAuth 2.0 scopes work coming. @mickmister - We're going to move forward and merge this PR. This content can be updated iteratively as needed if you continue to be concerned.